### ISSUE TITLE
[Legitimate Site Unblocked]

### ISSUE THREAD

**Issue by musacanminaz** [MetaMask/eth-phishing-detect#13563]
**Domain:** bnbplus.finance

**Details:** https://t.me/BNBplusfinance and https://x.com/bnbpluspasif?s=21&t=vWZhH7zrThXuvzjEDWI9fg these are social profiles this site gives bnb stake option and use wage web3modal and walletconnect last versions

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
i already asked for more information that was not provided on another issue. If you don't provide context, social accounts, etc on this issue in the next 24 hours this will be closed.

**Comment by musacanminaz** [comment]
I don't really understand what is required. on this site that may harm users' wallets.

Potential threats on https://bnbplus.finance/ include:

Fake versions of MetaMask
Secret Recovery Phrase or password theft
Malicious transactions resulting in stolen assets

We are sure that none of these writers exist. What is the relevance of social media accounts to the subject?

Telegram : https://t.me/BNBplusfinance
Twitter :  https://x.com/bnbpluspasif?s=21&t=vWZhH7zrThXuvzjEDWI9fg
